symbology/custom-dictionary-style/src/main/java/com/esri/samples/custom_dictionary_style/CustomDictionaryStyleSample.java

```diff
@@ -67,31 +92,120 @@ public void start(Stage stage) {
+        progressIndicator.setVisible(true);
+
+        if (toggleGroup.getSelectedToggle() == webStyleButton) {
+          dictionarySymbolStyle = new DictionarySymbolStyle(portalItem);
+          // load the symbol dictionary
+          dictionarySymbolStyle.loadAsync();
+          dictionarySymbolStyle.addDoneLoadingListener(() -> {
+            if (dictionarySymbolStyle.getLoadStatus() == LoadStatus.LOADED) {
+              // create a dictionary renderer with the dictionary symbol style
+              dictionaryRenderer = new DictionaryRenderer(dictionarySymbolStyle, fieldMap, fieldMap);
+              // renderer tells layer what symbols to apply to what features
+              restaurantLayer.setRenderer(dictionaryRenderer);
+              // add the layer to the map
+              map.getOperationalLayers().add(restaurantLayer);
+              // hide the progress indicator when the symbols have been applied to the features
+              progressIndicator.setVisible(false);
+            } else {
+              new Alert(Alert.AlertType.ERROR, "Dictionary symbol style failed to load!").show();
+            }
+          });
+        }
+        if (toggleGroup.getSelectedToggle() == fileStyleButton){
+          // create a symbol dictionary from the local symbol style file
+          dictionarySymbolStyle = DictionarySymbolStyle.createFromFile(stylxFile.getAbsolutePath());
+          // create a dictionary renderer with the dictionary symbol style
+          dictionaryRenderer = new DictionaryRenderer(dictionarySymbolStyle);
+          // renderer tells layer what symbols to apply to what features
+          restaurantLayer.setRenderer(dictionaryRenderer);
+          // add the layer to the map
+          map.getOperationalLayers().add(restaurantLayer);
+          // hide the progress indicator when the symbols have been applied to the features
+          progressIndicator.setVisible(false);
+        }
+      });
+
+      // and the control panel, map view, and progress indicator to the stack pane
+      stackPane.getChildren().addAll(mapView, controlsVBox, progressIndicator);
+      StackPane.setAlignment(controlsVBox, Pos.TOP_LEFT);
+      StackPane.setAlignment(progressIndicator, Pos.CENTER);
+      StackPane.setMargin(controlsVBox, new Insets(10, 0, 0, 10));
 
     } catch (Exception e) {
       e.printStackTrace();
     }
   }
 
+  /**
+   * Sets up control panel with radio buttons to toggle between the dictionary symbol styles.
+   */
+  public void setupUI() {
+    // create a label
+    Label symbolStyleLabel = new Label("Choose a Dictionary Symbol Style:");
+
+    // create radio buttons for toggling between the web and file styles
+    webStyleButton = new RadioButton("Web style");
+    fileStyleButton = new RadioButton("Style file ");
+
+    // set the radio buttons to a toggle group
+    toggleGroup = new ToggleGroup();
+    webStyleButton.setToggleGroup(toggleGroup);
+    fileStyleButton.setToggleGroup(toggleGroup);
```

Comment:
Perhaps we can add a line here to select one of the things by default? I found it a little odd that the sample loads up but none is selected, having to select one manually.